[
    "y/core -Inumpy/core/src/npymath -Inumpy/core/src/multiarray -Inump\n             y/core/src/umath -Inumpy/core/src/npysort -I/p/vast1/koparasy/BUIL\n             D/spack_cluster_installer/quartz/py-libensemble/spack_env_dir/linu\n             x-rhel7-broadwell/gcc-9.3.1/python-3.9.9-3o76m4utiyrsvuxwx2ytcv7sd\n             3g35si6/include/python3.9 -c'\n     830     gcc: _configtest.c\n831     _configtest.c:3: warning: ignoring #pragma clang diagnostic [-Wunk\n             nown-pragmas]\n     832         3 | #pragma clang diagnostic error \"-Wattributes\"\n     833           |\n     834     _configtest.c: In function '__declspec':\n  >> 835     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'",
    "836         5 | int __declspec(thread) foo;\n     837           |                        ^~~\n  >> 838     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     839         9 | {\n     840           | ^\n841     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     842         5 | int __declspec(thread) foo;\n     843           |     ^~~~~~~~~~\n  >> 844     _configtest.c:11: error: expected '{' at end of input\n     845        11 | }\n     846           |",
    "implicit-int]\n     842         5 | int __declspec(thread) foo;\n     843           |     ^~~~~~~~~~\n  >> 844     _configtest.c:11: error: expected '{' at end of input\n     845        11 | }\n     846           |\n847     _configtest.c:11: warning: control reaches end of non-void functio\n             n [-Wreturn-type]\n     848        11 | }\n     849           |\n     850     _configtest.c:3: warning: ignoring #pragma clang diagnostic [-Wunk\n             nown-pragmas]\n     851         3 | #pragma clang diagnostic error \"-Wattributes\"",
    "845        11 | }\n     846           |\n     847     _configtest.c:11: warning: control reaches end of non-void functio\n             n [-Wreturn-type]\n     848        11 | }\n     849           |\n850     _configtest.c:3: warning: ignoring #pragma clang diagnostic [-Wunk\n             nown-pragmas]\n     851         3 | #pragma clang diagnostic error \"-Wattributes\"\n     852           |\n     853     _configtest.c: In function '__declspec':\n  >> 854     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'",
    "855         5 | int __declspec(thread) foo;\n     856           |                        ^~~\n  >> 857     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     858         9 | {\n     859           | ^\n860     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     861         5 | int __declspec(thread) foo;\n     862           |     ^~~~~~~~~~\n  >> 863     _configtest.c:11: error: expected '{' at end of input\n     864        11 | }\n     865           |",
    "implicit-int]\n     861         5 | int __declspec(thread) foo;\n     862           |     ^~~~~~~~~~\n  >> 863     _configtest.c:11: error: expected '{' at end of input\n     864        11 | }\n     865           |\n866     _configtest.c:11: warning: control reaches end of non-void functio\n             n [-Wreturn-type]\n     867        11 | }\n     868           |\n     869     failure.\n\n     ...",
    "...\n\n     2942          |                              ^~\n     2943    numpy/core/src/multiarray/scalartypes.c.src: In function 'timedelt\n             a_arrtype_new':\n2944    numpy/core/src/multiarray/scalartypes.c.src:2707:30: warning: comp\n             arison of integer expressions of different signedness: 'NPY_DATETI\n             MEUNIT' {aka 'enum <anonymous>'} and 'int' [-Wsign-compare]\n     2945     2707 |         if (ret->obmeta.base == -1) {\n     2946          |                              ^~\n     2947    numpy/core/src/multiarray/scalartypes.c.src: In function 'initiali\n             ze_numeric_types':",
    "eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2967     4193 |     PyCLongDoubleArrType_Type.tp_print = clongdoubletype_p\n             rint;\n     2968          |                              ^\n     2969    numpy/core/src/multiarray/scalartypes.c.src: In function 'unicodet\n             ype_repr':\n2970    numpy/core/src/multiarray/scalartypes.c.src:594:5: warning: 'PyUni\n             code_AsUnicode' is deprecated [-Wdeprecated-declarations]\n     2971      594 |     ip = dptr = Py@Name@_AS_@NAME@(self);\n     2972          |     ^~\n\n     ...",
    "...\n\n     3109          |                              ^~\n     3110    numpy/core/src/multiarray/scalartypes.c.src: In function 'timedelt\n             a_arrtype_new':\n3111    numpy/core/src/multiarray/scalartypes.c.src:2707:30: warning: comp\n             arison of integer expressions of different signedness: 'NPY_DATETI\n             MEUNIT' {aka 'enum <anonymous>'} and 'int' [-Wsign-compare]\n     3112     2707 |         if (ret->obmeta.base == -1) {\n     3113          |                              ^~\n     3114    numpy/core/src/multiarray/scalartypes.c.src: In function 'initiali\n             ze_numeric_types':",
    "st.o -o _configtest\n     76      /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 77      /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.13.3-vsbutjuqkzwj\n             kiuc642bhggakyypewtj/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 78      collect2: error: ld returned 1 exit status\n     79      /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 80      /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.13.3-vsbutjuqkzwj\n             kiuc642bhggakyypewtj/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n  >> 81      collect2: error: ld returned 1 exit status",
    ">> 78      collect2: error: ld returned 1 exit status\n     79      /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 80      /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.13.3-vsbutjuqkzwj\n             kiuc642bhggakyypewtj/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 81      collect2: error: ld returned 1 exit status\n     82      failure.\n     83      removing: _configtest.c _configtest.o\n     84      C compiler: /p/vast1/koparasy/BUILD/head/spack/lib/spack/env/gcc/g\n             cc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall\n              -fPIC -fPIC\n     85",
    "y/core -Inumpy/core/src/npymath -Inumpy/core/src/multiarray -Inump\n             y/core/src/umath -Inumpy/core/src/npysort -I/p/vast1/koparasy/BUIL\n             D/spack_cluster_installer/quartz/py-libensemble/spack_env_dir/linu\n             x-rhel7-broadwell/gcc-9.3.1/python-3.9.9-3o76m4utiyrsvuxwx2ytcv7sd\n             3g35si6/include/python3.9 -c'\n     134     gcc: _configtest.c\n>> 135     _configtest.c:1:10: fatal error: sys/endian.h: No such file or dir\n             ectory\n     136         1 | #include <sys/endian.h>\n     137           |          ^~~~~~~~~~~~~~\n     138     compilation terminated.\n  >> 139     _configtest.c:1:10: fatal error: sys/endian.h: No such file or dir\n             ectory",
    "134     gcc: _configtest.c\n  >> 135     _configtest.c:1:10: fatal error: sys/endian.h: No such file or dir\n             ectory\n     136         1 | #include <sys/endian.h>\n     137           |          ^~~~~~~~~~~~~~\n     138     compilation terminated.\n>> 139     _configtest.c:1:10: fatal error: sys/endian.h: No such file or dir\n             ectory\n     140         1 | #include <sys/endian.h>\n     141           |          ^~~~~~~~~~~~~~\n     142     compilation terminated.\n     143     failure.\n     144     removing: _configtest.c _configtest.o",
    "y/core/src/umath -Inumpy/core/src/npysort -I/p/vast1/koparasy/BUIL\n             D/spack_cluster_installer/quartz/py-libensemble/spack_env_dir/linu\n             x-rhel7-broadwell/gcc-9.3.1/python-3.9.9-3o76m4utiyrsvuxwx2ytcv7sd\n             3g35si6/include/python3.9 -c'\n     250     gcc: _configtest.c\n     251     _configtest.c: In function 'main':\n>> 252     _configtest.c:7:12: error: 'SIZEOF_LONGDOUBLE' undeclared (first u\n             se in this function); did you mean 'SIZEOF_LONG_DOUBLE'?\n     253         7 |     (void) SIZEOF_LONGDOUBLE;\n     254           |            ^~~~~~~~~~~~~~~~~\n     255           |            SIZEOF_LONG_DOUBLE\n     256     _configtest.c:7:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in",
    "253         7 |     (void) SIZEOF_LONGDOUBLE;\n     254           |            ^~~~~~~~~~~~~~~~~\n     255           |            SIZEOF_LONG_DOUBLE\n     256     _configtest.c:7:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in\n     257     _configtest.c: In function 'main':\n>> 258     _configtest.c:7:12: error: 'SIZEOF_LONGDOUBLE' undeclared (first u\n             se in this function); did you mean 'SIZEOF_LONG_DOUBLE'?\n     259         7 |     (void) SIZEOF_LONGDOUBLE;\n     260           |            ^~~~~~~~~~~~~~~~~\n     261           |            SIZEOF_LONG_DOUBLE\n     262     _configtest.c:7:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in",
    "st.o -o _configtest\n     396     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 397     /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.13.3-vsbutjuqkzwj\n             kiuc642bhggakyypewtj/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 398     collect2: error: ld returned 1 exit status\n     399     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 400     /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.13.3-vsbutjuqkzwj\n             kiuc642bhggakyypewtj/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n  >> 401     collect2: error: ld returned 1 exit status",
    ">> 398     collect2: error: ld returned 1 exit status\n     399     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 400     /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.13.3-vsbutjuqkzwj\n             kiuc642bhggakyypewtj/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 401     collect2: error: ld returned 1 exit status\n     402     failure.\n     403     removing: _configtest.c _configtest.o\n     404     C compiler: /p/vast1/koparasy/BUILD/head/spack/lib/spack/env/gcc/g\n             cc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall\n              -fPIC -fPIC\n     405",
    "830     gcc: _configtest.c\n     831     _configtest.c:3: warning: ignoring #pragma clang diagnostic [-Wunk\n             nown-pragmas]\n     832         3 | #pragma clang diagnostic error \"-Wattributes\"\n     833           |\n     834     _configtest.c: In function '__declspec':\n>> 835     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'\n     836         5 | int __declspec(thread) foo;\n     837           |                        ^~~\n  >> 838     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     839         9 | {",
    "833           |\n     834     _configtest.c: In function '__declspec':\n  >> 835     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'\n     836         5 | int __declspec(thread) foo;\n     837           |                        ^~~\n>> 838     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     839         9 | {\n     840           | ^\n     841     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     842         5 | int __declspec(thread) foo;",
    "839         9 | {\n     840           | ^\n     841     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     842         5 | int __declspec(thread) foo;\n     843           |     ^~~~~~~~~~\n>> 844     _configtest.c:11: error: expected '{' at end of input\n     845        11 | }\n     846           |\n     847     _configtest.c:11: warning: control reaches end of non-void functio\n             n [-Wreturn-type]\n     848        11 | }\n     849           |",
    "849           |\n     850     _configtest.c:3: warning: ignoring #pragma clang diagnostic [-Wunk\n             nown-pragmas]\n     851         3 | #pragma clang diagnostic error \"-Wattributes\"\n     852           |\n     853     _configtest.c: In function '__declspec':\n>> 854     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'\n     855         5 | int __declspec(thread) foo;\n     856           |                        ^~~\n  >> 857     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     858         9 | {",
    "852           |\n     853     _configtest.c: In function '__declspec':\n  >> 854     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'\n     855         5 | int __declspec(thread) foo;\n     856           |                        ^~~\n>> 857     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     858         9 | {\n     859           | ^\n     860     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     861         5 | int __declspec(thread) foo;",
    "858         9 | {\n     859           | ^\n     860     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     861         5 | int __declspec(thread) foo;\n     862           |     ^~~~~~~~~~\n>> 863     _configtest.c:11: error: expected '{' at end of input\n     864        11 | }\n     865           |\n     866     _configtest.c:11: warning: control reaches end of non-void functio\n             n [-Wreturn-type]\n     867        11 | }\n     868           |",
    "y/core/src/umath -Inumpy/core/src/npysort -I/p/vast1/koparasy/BUIL\n             D/spack_cluster_installer/quartz/py-libensemble/spack_env_dir/linu\n             x-rhel7-broadwell/gcc-9.3.1/python-3.9.9-3o76m4utiyrsvuxwx2ytcv7sd\n             3g35si6/include/python3.9 -c'\n     1196    gcc: _configtest.c\n     1197    _configtest.c: In function 'main':\n>> 1198    _configtest.c:8:12: error: 'HAVE_DECL_SIGNBIT' undeclared (first u\n             se in this function); did you mean 'HAVE_DECL_ISNAN'?\n     1199        8 |     (void) HAVE_DECL_SIGNBIT;\n     1200          |            ^~~~~~~~~~~~~~~~~\n     1201          |            HAVE_DECL_ISNAN\n     1202    _configtest.c:8:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in",
    "1199        8 |     (void) HAVE_DECL_SIGNBIT;\n     1200          |            ^~~~~~~~~~~~~~~~~\n     1201          |            HAVE_DECL_ISNAN\n     1202    _configtest.c:8:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in\n     1203    _configtest.c: In function 'main':\n>> 1204    _configtest.c:8:12: error: 'HAVE_DECL_SIGNBIT' undeclared (first u\n             se in this function); did you mean 'HAVE_DECL_ISNAN'?\n     1205        8 |     (void) HAVE_DECL_SIGNBIT;\n     1206          |            ^~~~~~~~~~~~~~~~~\n     1207          |            HAVE_DECL_ISNAN\n     1208    _configtest.c:8:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in",
    "st.o -o _configtest\n     1790    /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 1791    /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.13.3-vsbutjuqkzwj\n             kiuc642bhggakyypewtj/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 1792    collect2: error: ld returned 1 exit status\n     1793    /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 1794    /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.13.3-vsbutjuqkzwj\n             kiuc642bhggakyypewtj/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n  >> 1795    collect2: error: ld returned 1 exit status",
    ">> 1792    collect2: error: ld returned 1 exit status\n     1793    /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 1794    /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.13.3-vsbutjuqkzwj\n             kiuc642bhggakyypewtj/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 1795    collect2: error: ld returned 1 exit status\n     1796    failure.\n     1797    removing: _configtest.c _configtest.o\n     1798    C compiler: /p/vast1/koparasy/BUILD/head/spack/lib/spack/env/gcc/g\n             cc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall\n              -fPIC -fPIC\n     1799",
    "arison of integer expressions of different signedness: 'NPY_DATETI\n             MEUNIT' {aka 'enum <anonymous>'} and 'int' [-Wsign-compare]\n     2945     2707 |         if (ret->obmeta.base == -1) {\n     2946          |                              ^~\n     2947    numpy/core/src/multiarray/scalartypes.c.src: In function 'initiali\n             ze_numeric_types':\n>> 2948    numpy/core/src/multiarray/scalartypes.c.src:4186:23: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2949     4186 |     PyHalfArrType_Type.tp_print = halftype_print;\n     2950          |                       ^\n  >> 2951    numpy/core/src/multiarray/scalartypes.c.src:4187:24: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2952     4187 |     PyFloatArrType_Type.tp_print = floattype_print;",
    "2947    numpy/core/src/multiarray/scalartypes.c.src: In function 'initiali\n             ze_numeric_types':\n  >> 2948    numpy/core/src/multiarray/scalartypes.c.src:4186:23: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2949     4186 |     PyHalfArrType_Type.tp_print = halftype_print;\n     2950          |                       ^\n>> 2951    numpy/core/src/multiarray/scalartypes.c.src:4187:24: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2952     4187 |     PyFloatArrType_Type.tp_print = floattype_print;\n     2953          |                        ^\n  >> 2954    numpy/core/src/multiarray/scalartypes.c.src:4188:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2955     4188 |     PyDoubleArrType_Type.tp_print = doubletype_print;",
    "2949     4186 |     PyHalfArrType_Type.tp_print = halftype_print;\n     2950          |                       ^\n  >> 2951    numpy/core/src/multiarray/scalartypes.c.src:4187:24: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2952     4187 |     PyFloatArrType_Type.tp_print = floattype_print;\n     2953          |                        ^\n>> 2954    numpy/core/src/multiarray/scalartypes.c.src:4188:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2955     4188 |     PyDoubleArrType_Type.tp_print = doubletype_print;\n     2956          |                         ^\n  >> 2957    numpy/core/src/multiarray/scalartypes.c.src:4189:29: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2958     4189 |     PyLongDoubleArrType_Type.tp_print = longdoubletype_pri",
    "2952     4187 |     PyFloatArrType_Type.tp_print = floattype_print;\n     2953          |                        ^\n  >> 2954    numpy/core/src/multiarray/scalartypes.c.src:4188:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2955     4188 |     PyDoubleArrType_Type.tp_print = doubletype_print;\n     2956          |                         ^\n>> 2957    numpy/core/src/multiarray/scalartypes.c.src:4189:29: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2958     4189 |     PyLongDoubleArrType_Type.tp_print = longdoubletype_pri\n             nt;\n     2959          |                             ^\n  >> 2960    numpy/core/src/multiarray/scalartypes.c.src:4191:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'",
    "2956          |                         ^\n  >> 2957    numpy/core/src/multiarray/scalartypes.c.src:4189:29: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2958     4189 |     PyLongDoubleArrType_Type.tp_print = longdoubletype_pri\n             nt;\n     2959          |                             ^\n>> 2960    numpy/core/src/multiarray/scalartypes.c.src:4191:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2961     4191 |     PyCFloatArrType_Type.tp_print = cfloattype_print;\n     2962          |                         ^\n  >> 2963    numpy/core/src/multiarray/scalartypes.c.src:4192:26: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2964     4192 |     PyCDoubleArrType_Type.tp_print = cdoubletype_print;",
    "nt;\n     2959          |                             ^\n  >> 2960    numpy/core/src/multiarray/scalartypes.c.src:4191:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2961     4191 |     PyCFloatArrType_Type.tp_print = cfloattype_print;\n     2962          |                         ^\n>> 2963    numpy/core/src/multiarray/scalartypes.c.src:4192:26: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2964     4192 |     PyCDoubleArrType_Type.tp_print = cdoubletype_print;\n     2965          |                          ^\n  >> 2966    numpy/core/src/multiarray/scalartypes.c.src:4193:30: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2967     4193 |     PyCLongDoubleArrType_Type.tp_print = clongdoubletype_p",
    "2961     4191 |     PyCFloatArrType_Type.tp_print = cfloattype_print;\n     2962          |                         ^\n  >> 2963    numpy/core/src/multiarray/scalartypes.c.src:4192:26: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2964     4192 |     PyCDoubleArrType_Type.tp_print = cdoubletype_print;\n     2965          |                          ^\n>> 2966    numpy/core/src/multiarray/scalartypes.c.src:4193:30: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2967     4193 |     PyCLongDoubleArrType_Type.tp_print = clongdoubletype_p\n             rint;\n     2968          |                              ^\n     2969    numpy/core/src/multiarray/scalartypes.c.src: In function 'unicodet\n             ype_repr':",
    "arison of integer expressions of different signedness: 'NPY_DATETI\n             MEUNIT' {aka 'enum <anonymous>'} and 'int' [-Wsign-compare]\n     3112     2707 |         if (ret->obmeta.base == -1) {\n     3113          |                              ^~\n     3114    numpy/core/src/multiarray/scalartypes.c.src: In function 'initiali\n             ze_numeric_types':\n>> 3115    numpy/core/src/multiarray/scalartypes.c.src:4186:23: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3116     4186 |     PyHalfArrType_Type.tp_print = halftype_print;\n     3117          |                       ^\n  >> 3118    numpy/core/src/multiarray/scalartypes.c.src:4187:24: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3119     4187 |     PyFloatArrType_Type.tp_print = floattype_print;",
    "3114    numpy/core/src/multiarray/scalartypes.c.src: In function 'initiali\n             ze_numeric_types':\n  >> 3115    numpy/core/src/multiarray/scalartypes.c.src:4186:23: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3116     4186 |     PyHalfArrType_Type.tp_print = halftype_print;\n     3117          |                       ^\n>> 3118    numpy/core/src/multiarray/scalartypes.c.src:4187:24: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3119     4187 |     PyFloatArrType_Type.tp_print = floattype_print;\n     3120          |                        ^\n  >> 3121    numpy/core/src/multiarray/scalartypes.c.src:4188:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3122     4188 |     PyDoubleArrType_Type.tp_print = doubletype_print;",
    "3116     4186 |     PyHalfArrType_Type.tp_print = halftype_print;\n     3117          |                       ^\n  >> 3118    numpy/core/src/multiarray/scalartypes.c.src:4187:24: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3119     4187 |     PyFloatArrType_Type.tp_print = floattype_print;\n     3120          |                        ^\n>> 3121    numpy/core/src/multiarray/scalartypes.c.src:4188:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3122     4188 |     PyDoubleArrType_Type.tp_print = doubletype_print;\n     3123          |                         ^\n  >> 3124    numpy/core/src/multiarray/scalartypes.c.src:4189:29: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3125     4189 |     PyLongDoubleArrType_Type.tp_print = longdoubletype_pri",
    "3119     4187 |     PyFloatArrType_Type.tp_print = floattype_print;\n     3120          |                        ^\n  >> 3121    numpy/core/src/multiarray/scalartypes.c.src:4188:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3122     4188 |     PyDoubleArrType_Type.tp_print = doubletype_print;\n     3123          |                         ^\n>> 3124    numpy/core/src/multiarray/scalartypes.c.src:4189:29: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3125     4189 |     PyLongDoubleArrType_Type.tp_print = longdoubletype_pri\n             nt;\n     3126          |                             ^\n  >> 3127    numpy/core/src/multiarray/scalartypes.c.src:4191:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'",
    "3123          |                         ^\n  >> 3124    numpy/core/src/multiarray/scalartypes.c.src:4189:29: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3125     4189 |     PyLongDoubleArrType_Type.tp_print = longdoubletype_pri\n             nt;\n     3126          |                             ^\n>> 3127    numpy/core/src/multiarray/scalartypes.c.src:4191:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3128     4191 |     PyCFloatArrType_Type.tp_print = cfloattype_print;\n     3129          |                         ^\n  >> 3130    numpy/core/src/multiarray/scalartypes.c.src:4192:26: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3131     4192 |     PyCDoubleArrType_Type.tp_print = cdoubletype_print;",
    "nt;\n     3126          |                             ^\n  >> 3127    numpy/core/src/multiarray/scalartypes.c.src:4191:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3128     4191 |     PyCFloatArrType_Type.tp_print = cfloattype_print;\n     3129          |                         ^\n>> 3130    numpy/core/src/multiarray/scalartypes.c.src:4192:26: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3131     4192 |     PyCDoubleArrType_Type.tp_print = cdoubletype_print;\n     3132          |                          ^\n  >> 3133    numpy/core/src/multiarray/scalartypes.c.src:4193:30: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3134     4193 |     PyCLongDoubleArrType_Type.tp_print = clongdoubletype_p",
    "3128     4191 |     PyCFloatArrType_Type.tp_print = cfloattype_print;\n     3129          |                         ^\n  >> 3130    numpy/core/src/multiarray/scalartypes.c.src:4192:26: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3131     4192 |     PyCDoubleArrType_Type.tp_print = cdoubletype_print;\n     3132          |                          ^\n>> 3133    numpy/core/src/multiarray/scalartypes.c.src:4193:30: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     3134     4193 |     PyCLongDoubleArrType_Type.tp_print = clongdoubletype_p\n             rint;\n     3135          |                              ^\n  >> 3136    error: Command \"/p/vast1/koparasy/BUILD/head/spack/lib/spack/env/g\n             cc/gcc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -"
]